### PR TITLE
Fix faulty URLs that lead to 404

### PR DESCRIPTION
**Pull request by padmkris123**
Fix faulty URLs that lead to 404

## How Has This Been Tested?

Tested it locally. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/uxp_for_you/uxp_for_newbies/index.md [hunk 1]
```diff
@@ -27,6 +27,6 @@ Don't be discouraged. One popular starting point is the [Mozilla Developers Netw
 
 The same applies to learning HTML and CSS (Cascading Style Sheets), although both of those are arguably easier than JavaScript. Don't go too far learning any of these, as the HTML, CSS, and JavaScript you need to know to use UXP is much simpler than what's required for a typical website.
 
-An apocryphal quote from Picasso is "good artists borrow, great artists steal" -- which means that, when you get to the point where you're ready to try coding, looking at sample code and putting it in your own projects is usually just fine. Then go through the code at your leisure and make sure you understand what it's doing. And to start you off with some sample code, [look here](../../../code_samples/).
+An apocryphal quote from Picasso is "good artists borrow, great artists steal" -- which means that, when you get to the point where you're ready to try coding, looking at sample code and putting it in your own projects is usually just fine. Then go through the code at your leisure and make sure you understand what it's doing. And to start you off with some sample code, [look here](https://github.com/AdobeDocs/uxp-photoshop-plugin-samples).
```
Comment (amandahuarng) [inline src/pages/guides/uxp_for_you/uxp_for_newbies/index.md:30]: So we don't want to link to this page (https://developer.adobe.com/photoshop/uxp/2022/guides/code_samples/) here instead?

**Review by amandahuarng — COMMENTED** [review]
Just one comment - otherwise lgtm

File: src/pages/guides/uxp_for_you/uxp_for_newbies/index.md (hunk 1 above)
Comment (padmkris123) [reply src/pages/guides/uxp_for_you/uxp_for_newbies/index.md:30]: I felt that based on the description we might want to take them to full-fledged plugin samples.

Comment (amandahuarng) [reply src/pages/guides/uxp_for_you/uxp_for_newbies/index.md:30]: Makes sense. 👍

**Review by amandahuarng — APPROVED** [review]

**Pull request merged**